modules/delegation-renewal/hooks/useDelegatedToExpired.tsx

```diff
@@ -0,0 +1,23 @@
+import { add, isBefore } from 'date-fns';
+import useDelegatedTo from 'modules/delegates/hooks/useDelegatedTo';
+import { useActiveWeb3React } from 'modules/web3/hooks/useActiveWeb3React';
+
+export default function useDelegatedToExpired(): boolean {
+  const { account: address, network } = useActiveWeb3React();
+
+  const { data: delegatedToData } = useDelegatedTo(address, network);
+
+  const isADelegateAboutToExpire = delegatedToData
+    ? delegatedToData.delegatedTo.reduce((prev, next) => {
+        // check if it has less than one week to expire or is expired
+        const isExpiredOrAboutToExpire = isBefore(
+          new Date(next.expirationDate),
+          add(new Date(), { days: 7 })
```

Comment:
Here we have 7 days but on the `AddressDelegatedTo` we use 30 days for about to expire. We should figure out which one to use, and then maybe make a constant in the app we can use so that if we update it, we just need to change the days in one place.

Reply:
I thought the banner gives a sense of more urgency, and in the list of addresses delegating like you have more time to visualize that the problem is coming. We should ask deniz